### DISCUSSION TITLE
[HELP] No rumble N64

### DISCUSSION THREAD

**Discussion by h4x0rJyNx** [darthcloud/BlueRetro#593] [Q&A]
Here is what I am using

Retroscaler Bluererto N64 single port adapter
Retro Fighters Brawler 64 Bluethooth controller
8Bitdo SN30 Pro Bluethooth controller

I am not connecting both controllers at the same time only one at a time. I have made sure on multiple times with a controller testing rom that I am pressing the correct button to switch between rumble pak and memory card. this also shows me that is switched and has a function to vibrate the controller. I have used an orignal N64 wired controller and a rumble pak and it does rumble. I read through as many things as I could find here on things to try. Has anyone eles had any issues like this?

**Comment by darthcloud** [comment]
In general rumble only work with 1st party device coming from Nintendo, PlayStation or Xbox.

This is because 3rd party device often only implement a subset of the possible rumble effect.
BlueRetro only implement a subset aswell so most of the time the subset implemented in both BlueRetro and 3rd controller do not overlap.

This will hopefully improve in the future but currently this is it.

**Reply by h4x0rJyNx** [reply]
That makes sense. What is needed to improve this? Just curious and I know it's probably not too high on the list of things to add.
